com.deflemask.DefleMask.yaml

```diff
@@ -0,0 +1,39 @@
+%YAML 1.1
```

Comment:
Do you actually need that for anything but technical correctness?

Reply:
nope, should I remove it?

Reply:
I don't think any other app uses it, flatpak-builder will work fine without it.